specification/maps/resource-manager/Microsoft.Maps/stable/2021-02-01/examples/ListKeys.json

```diff
@@ -0,0 +1,17 @@
+{
+  "parameters": {
+    "api-version": "2021-02-01",
+    "subscriptionId": "21a9967a-e8a9-4656-a70b-96ff1c4d05a0",
+    "resourceGroupName": "myResourceGroup",
+    "accountName": "myMapsAccount"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/21a9967a-e8a9-4656-a70b-96ff1c4d05a0/resourceGroups/myResourceGroup/providers/Microsoft.Maps/accounts/myMapsAccount",
```

Comment:
not sure why you need this in the response - since the request url contains it too

Reply:
Good point, I also noticed our schema is missing 2 properties. Will remove id and add missing properties.

Reply:
Fixed and added missing properties.